WireNetwork/Sources/WireNetwork/APIs/SelfUserAPI/SelfUserAPIV0.swift

```diff
@@ -67,6 +67,23 @@ class SelfUserAPIV0: SelfUserAPI, VersionedAPI {
         return try ResponseParser()
             .success(code: .ok)
             .parse(code: response.statusCode, data: data)
+
+    }
+
+    func deleteTeam(teamId: UUID, password: String, verificationCode: String) async throws {
+        let body = try JSONEncoder.defaultEncoder.encode(
+            DeleteTeamRequestBodyV0(password: password, verificationCode: verificationCode)
+        )
+
+        let request = try URLRequestBuilder(path: "/v8/teams/\(teamId)")
```

Comment:
issue: prefix is set in V0 class, 

```suggestion
        let request = try URLRequestBuilder(path: "\(pathPrefix)/teams/\(teamId)")
```

Reply:
done